app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,92 @@
+require 'sinatra/base'
+require 'rack-flash'
+class TweetController < ApplicationController
+
+    enable :sessions
+    enable :method_override
+
+    use Rack::Flash
+ 
+    get '/tweets' do
+        if logged_in?(session)
+            @user = current_user(session)
+            @tweets = Tweet.all 
+            @users = User.all
+        else
+            redirect to "/login"
+        end
+        erb :'tweets/tweets'
+    end
+
+    get '/tweets/new' do
+        if logged_in?(session)
+            erb :'tweets/create_tweet'
+        else
+            redirect to "/login"
+        end
+    end
+
+    post '/tweets' do
+        @tweet = Tweet.new(params)
```

Comment:
The tweet creation and user association can both be accomplished in the same step:

``` ruby
@tweet = Tweet.create(content: params[:content], user: current_user)
```